### ISSUE TITLE
Budgie Menu search bar

### ISSUE THREAD

**Issue by tristan957** [system76/pop-gtk-theme#27]
```
 * Distribution - Solus
 * Gtk+ 3.0 version - 3.22
 * Desktop environment - Budgie
 ```

**Pop version (if you knew)**
1.3.1 r1

**Related Application and/or Shell theming (or name of widget)**
Not sure

**Actual issue**
The search area is a different color than the rest of the menu.
![budgie-menu](https://cloud.githubusercontent.com/assets/11936772/26765347/f93ab400-493f-11e7-98da-a7ec93973c71.png)

**Steps to reproduce (if you knew)**
Open the menu

FYI I have this building on Solus just fine. Making strides to get this packaged in the main repo.

**Comment by isantop** [comment]
Thanks for letting us know about this! It looks like it's using the standard GtkEntry styling, which applies an F6F6F6 colored background with 10% opacity. We do this to ensure that the entries are easily distinguished as entries when they're rendered in headerbars (since they aren't clickable) and to add some distinction to them in the dark theme. I would agree that it looks weird there though.

Can you look at what the name of that widget is? I can apply different styling to it but I'll need its name (The GTK property) in order to ensure it only applies to that widget specifically. You may be able to get the name through the GTK Inspector.

**Comment by tristan957** [comment]
I'll try to find this out for you. I didn't find anything looking through budgie specific theming in 3rdparty.scss although I could have missed it.

**Comment by isantop** [comment]
That's correct; Pop isn't applying any entry-specific theming to the Budgie menu, so it's using the theme default. You can also see an example of this (with the intended design) in the headerbar of GNOME Maps.
